### PR TITLE
Chocolatey deployment

### PR DESCRIPTION
**Pull request by fcastilloec**
This PR addresses issue #124 
We only need to setup `CHOCO_TOKEN` as an environmental variable on AppVeyor for it to be deployed.

### CODE REVIEW

File: windows/chocolatey/tools/chocolateyInstall.ps1 [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+$url          = "https://github.com/headsetapp/headset-electron/releases/download/v$version/HeadsetSetup.exe"
+$autoStartKey = 'HKLM:\SOFTWARE\Wow6432Node\Microsoft\Windows\CurrentVersion\Run\'
+$startKey     = 'HKLM:\SOFTWARE\Microsoft\Windows\CurrentVersion\Explorer\StartupApproved\Run32'
+
+$packageArgs = @{
+  packageName    = 'headset'
+  softwareName   = 'Headset'
+  fileType       = 'exe'
+  url            = $url
+  checksum       = 'NEEDS_TO_BE_REPLACED_BY_APPVEYOR!'
```
Comment (danielravina) [inline windows/chocolatey/tools/chocolateyInstall.ps1]: is this a placeholder?

**Comment by danielravina** [comment]
`CHOCO_TOKEN` added

File: windows/chocolatey/tools/chocolateyInstall.ps1 (hunk 1 above)
Comment (fcastilloec) [reply windows/chocolatey/tools/chocolateyInstall.ps1]: Yes, this line gets changed by AppVeyor when we run `build_choco.ps1`. That script gets the `sha256sum` of the signed `exe` and puts the value there, and then builds the chocolatey packages.

**Comment by fcastilloec** [comment]
Actually, I realized that I'm missing one more script, one that will change the version number of the chocolatey release file when we release a new tag/version. So we have to wait a little bit before merging and releasing

**Pull request merged**